### PR TITLE
singledispatch only needed for < 3.4

### PR DESCRIPTION
**Pull request by dwoz**
### Commits signed with GPG?

Yes

### CODE REVIEW

File: requirements/static/py3.5/linux.txt [hunk 1]
```diff
@@ -105,8 +105,7 @@ salttesting==2017.6.1
 scp==0.13.2               # via junos-eznc
 setproctitle==1.1.10
 setuptools-scm==3.2.0
-singledispatch==3.4.0.3
```
Comment (waynew) [inline requirements/static/py3.5/linux.txt]: The title of this PR says it's not needed for < 3.4 - is that pointing the wrong way?

Comment (dwoz) [reply requirements/static/py3.5/linux.txt]: Yeah, the commit message was wrong. Fixed.

**Review by waynew — APPROVED** [review]

**Review by dmurphy18 — CHANGES_REQUESTED** [review]
backports_abc need the change too ?, I believe it does

**Comment by dwoz** [comment]
> backports_abc need the change too ?, I believe it does

Yes, fixed now.

**Review by s0undt3ch — APPROVED** [review]

**Review by dmurphy18 — APPROVED** [review]

**Pull request merged**